### PR TITLE
Third Header

### PR DESCRIPTION
**Pull request by witflash**
header_third

### CODE REVIEW

File: app/styles/headers/_third.scss [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+		justify-content: space-between;
+		height: 100%;
+
+		&--flex {
+			display: flex;
+		}
+
+		&--center {
+			margin-left: 20%;
+			margin-right: 20%;
```
Comment (Vitaliy-Maznyi) [inline app/styles/headers/_third.scss]: Can you use here `margin: auto` or it doesn't work?

File: app/styles/headers/_third.scss [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+}
+
+.menu a, .hamb-menu a {
+	text-decoration: none;
+	text-transform: uppercase;
+	color: $color--main;
+}
+
+.hamb-menu {
+	font-size: 130%;
```
Comment (Vitaliy-Maznyi) [inline app/styles/headers/_third.scss]: It's not very good to use values more than 100%. You can set it in `px` or `em`

File: app/styles/headers/_third.scss [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+	display: flex;
+	flex-wrap: wrap;
+	padding: 20px;
+	align-items: center;
+	background-color: $color--grey_lighten;
+	border-radius: 6px;
+	font-family: $font-family;
+
+	&__input {
+		width: 39%;
```
Comment (Vitaliy-Maznyi) [inline app/styles/headers/_third.scss]: why do you use here `39%`?

Comment (Vitaliy-Maznyi) [reply app/styles/headers/_third.scss]: maybe you can use here `flex: 1`. Try to find out how to use here flex

File: app/styles/headers/_third.scss [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+		padding-right: 20px;
+		box-sizing: border-box;
+	}
+
+	&__button {
+		width: 22%;
```
Comment (Vitaliy-Maznyi) [inline app/styles/headers/_third.scss]: you don't need to set button width. Buttons width should depends on their content. And button size you can control with padding

**Review by Vitaliy-Maznyi — DISMISSED** [review]

File: app/styles/headers/_third.scss (hunk 1 above)
Comment (witflash) [reply app/styles/headers/_third.scss]: It's may be:
`width: 60%;`
`margin: auto;`
I will try :)

File: app/styles/headers/_third.scss (hunk 2 above)
Comment (witflash) [reply app/styles/headers/_third.scss]: Accepted )

File: app/styles/headers/_third.scss (hunk 3 above)
Comment (witflash) [reply app/styles/headers/_third.scss]: Two inputs by 39% + one button by 22% = 100% :)
Is there maybe another way to line up three elements with fixed distance between them?
I'll think what can I do in this case.

File: app/styles/headers/_third.scss (hunk 4 above)
Comment (witflash) [reply app/styles/headers/_third.scss]: `form__input` & `form__button` are wrappers for blocks (`input` & `button`) for positioning in form's block. Button's block doesn't have fixed width.

Comment (witflash) [reply app/styles/headers/_third.scss]: Oh, I think, I understood what did you mean. I'll fix it.

**Comment by witflash** [comment]
I fixed all bugs.

**Pull request merged**